### PR TITLE
[wip] PHP >= 5.5, ZF ^2.5, zend-hydrator ~1.0

### PR DESCRIPTION
**Pull request by TomHAnderson**
* Replace references to Zend\Stdlib\Hydrator to Zend\Hydrator ~1.0; requires PHP >= 5.5
* Up composer Zend Framework to ^2.5

### CODE REVIEW

**Comment by TomHAnderson** [comment]
Before this PR can move forward a decision must be made for which version of Zend Framework to support.  https://github.com/doctrine/DoctrineMongoODMModule/blob/master/composer.json uses "2.*" which I think is a better tag to move forward with.

**Comment by TomHAnderson** [comment]
Correction, 2.* would break this backwards so ^2.5 is correct.

File: src/DoctrineModule/Stdlib/Hydrator/DoctrineObject.php [hunk 1]
```diff
@@ -27,8 +27,8 @@
 use RuntimeException;
 use Traversable;
 use Zend\Stdlib\ArrayUtils;
-use Zend\Stdlib\Hydrator\AbstractHydrator;
-use Zend\Stdlib\Hydrator\Filter\FilterProviderInterface;
+use Zend\Hydrator\AbstractHydrator;
```
Comment (Ocramius) [inline src/DoctrineModule/Stdlib/Hydrator/DoctrineObject.php]: @gianarb this is a major BC break that calls for a v2.0

**Comment by TomHAnderson** [comment]
@gianarb What's the status on this?

**Comment by TomHAnderson** [comment]
@gianarb What's the status on this?

**Comment by stavarengo** [comment]
+1

**Comment by jackdpeterson** [comment]
@Ocramius Can this get integrated/tagged for 2.0?

**Comment by TomHAnderson** [comment]
!ping @doctrinebot It's only been six months since this was proposed.  How much more time do you need to see create a new version to enable this BC Break?

**Comment by TomHAnderson** [comment]
Moving my chips to https://github.com/doctrine/DoctrineModule/pull/564

**Pull request closed without merging**